### DISCUSSION TITLE
Will the upcoming API changes with Bitwarden impact their client connectivity to vaultwarden servers?

### DISCUSSION THREAD

**Discussion by alphabet5** [dani-garcia/vaultwarden#2819] [Q&A]
> On November 16, 2022, Bitwarden will no longer be supporting the API related to self-hosted environments on versions 1.45 (Feb. 2022) and older.
> 
> To avoid disruption to service, please update your on-premise installation. If you have any questions, please contact the support team directly.

Will these API changes break bitwarden cilent connections to vaultwarden instances?

**Comment by BlackDex** [comment]
Vaultwarden is up-to-date with the latest releases.
All new client versions work with the latest Vaultwarden.

I'm not sure which specific items and endpoins will be deprecated from that date. But i think it shouldn't have an impact.
